app/views/submissions/_form.html.erb

```diff
@@ -63,8 +67,26 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" style="display:none"><i class="fas fa-check"></i> <%=t('submission.correct_file_size')%></div>
+			<div class="col-12" id="file-size-too-big" style="display:none"> <i class="fas fa-exclamation-triangle"></i> <%=t('submission.file_size_too_big')%></div>
+			<div class="col-12" id="file-size-way-too-big" style="display:none"><i class="fas fa-bomb"></i> <%=t('submission.file_size_way_too_big')%></div>
+			<div class="col-12" id="file-optimize" style="display:none">
+			<small><%=raw t("submission.optimization_help")%><%= helpdesk(t('submission.optimization_info'),true)%></small>
```

Comment:
I think the railsier way to do this would be `<%= t('submission.optimization_help_html') %>` instead of `<%=raw t("submission.optimization_help")%>` and then rename `submission.optimization_help` to `submission.optimization_help_html` in the locale files (see [here](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations)).